.circleci/config.yml

```diff
@@ -5,7 +5,7 @@
 
 defaults: &defaults
   docker:
-    - image: circleci/node:10.11-browsers
+    - image: circleci/node:10.15.3
```

Comment:
do we care if this gets out of sync with our other pieces?  I wouldn't think it would matter ...

Reply:
I figure it's like ruby versions in our CI builds. When it's time to bump node, we should do it across the board. For now, I'm at least trying to catch the pipeline up with `sinopia_acl` and `sinopia_user_backup`.

Reply:
gotcha.  And you don't need "browsers" here?

Reply:
@ndushay nopers! no browsers required. (was originally copypasta from the editor codebase, and I didn't understand the difference.)